pkg/utils/proxy/cloudproxy.go

```diff
@@ -174,3 +175,26 @@ func (cp *CloudProxy) container() (v1.Container, error) {
 func (cp *CloudProxy) buildConfigMap() (*v1.ConfigMap, error) {
 	return nil, nil
 }
+
+func (cp *CloudProxy) buildPromService() (*promv1.ServiceMonitor, error) {
```

Comment:
`buildServiceMonitor()` is more straightforward

Reply:
changed.